src/app/components/MnemonicInputs/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { wordlist } from "@scure/bip39/wordlists/english";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import Input from "~/app/components/form/Input";
+
+type MnemonicInputsProps = {
+  mnemonic?: string;
+  setMnemonic?(mnemonic: string): void;
+  disabled?: boolean;
+};
+
+export default function MnemonicInputs({
+  mnemonic,
+  setMnemonic,
+  disabled,
+  children,
+}: React.PropsWithChildren<MnemonicInputsProps>) {
+  const { t } = useTranslation("translation", {
+    keyPrefix: "accounts.account_view.mnemonic",
+  });
+  const [revealedIndex, setRevealedIndex] = useState<number | undefined>(
+    undefined
+  );
+
+  const words = mnemonic?.split(" ") || [];
+  while (words.length < 12) {
+    words.push("");
+  }
+
+  return (
+    <div className="border-[1px] border-gray-200 rounded-lg py-8 px-4 flex flex-col gap-8 items-center justify-center w-[520px] self-center">
+      <h3 className="font-semibold dark:text-white">{t("inputs.title")}</h3>
+      <div className="flex flex-wrap gap-4 justify-center items-center">
+        {[...new Array(12)].map((_, i) => (
+          <div key={i} className="flex justify-center items-center">
+            <span className="w-7 text-gray-500 slashed-zero dark:text-neutral-500">
+              {i + 1}.
+            </span>
+            <Input
+              type={revealedIndex === i ? "text" : "password"}
+              required
+              onFocus={() => setRevealedIndex(i)}
+              onBlur={() => setRevealedIndex(undefined)}
+              //disabled={disabled}
+              contentEditable={!disabled}
+              block={false}
+              className="w-24 text-center"
+              autoComplete=""
```

Comment:
do we really need this property to prevent password managers from saving / suggesting something? also test with firefox.

Reply:
removed `autoComplete` as it did nothing.

I also no longer set the type to password any more when the input is not selected because it had multiple issues on firefox (autocomplete not working, plus it kept suggesting passwords even using `autoComplete="off"`).

I think this is OK though because we change the value of the input to "•••••" when it is not selected, so the words cannot be cached by the browser